docs/content/stable/manage/upgrade-deployment.md

```diff
@@ -51,30 +53,40 @@ To upgrade YugabyteDB to a version based on a different version of PostgreSQL (f
 
 - You can upgrade from one stable version to another in one go, even across major versions, as long as they are in the same major YSQL version. For information on performing major YSQL version upgrades, refer to [YSQL major upgrade](../ysql-major-upgrade-yugabyted/).
 
+### Backups and point-in-time-recovery
+
 - Backups
 
   - Backups taken on a newer version cannot be restored to universes running a previous version.
   - Backups taken during the upgrade cannot be restored to universes running a previous version.
   - Backups taken before the upgrade _can_ be used for restore to the new version.
 
-- [Point-in-time-restore](../backup-restore/point-in-time-recovery/) (PITR)
+- [Point-in-time-recovery](../backup-restore/point-in-time-recovery/) (PITR)
```

Comment:
@hari90 @hulien22  Can you please check if this change is correct for OSS upgrade? i.e., while upgrading the OSS YBDB, the user doesn't need to delete the pitr config beforehand and YBDB can handle it as described?